efficientdet/inference.py

```diff
@@ -255,14 +255,14 @@ def det_post_process_combined(params, cls_outputs, box_outputs, scales,
   image_ids = tf.cast(
       tf.tile(
           tf.expand_dims(tf.range(batch_size), axis=1), [1, max_boxes_to_draw]),
-      dtype=tf.float32)
+      dtype=scales.dtype)
```

Comment:
Why?

Reply:
Oops, it was an old change. Reverted now.